src/main/java/ShuntingYardCalculator/Logging/Log4j.java

```diff
@@ -0,0 +1,45 @@
+package ShuntingYardCalculator.Logging;
```

Comment:
My meaning was if you use the function only once do not create a function for it
leave 2/3 function that know how to print logs in various situations